database/db.go

```diff
@@ -0,0 +1,74 @@
+package database
+
+import (
+	"fmt"
+	"github.com/go-pg/pg"
+	"github.com/robinjoseph08/go-pg-migrations"
+	"log"
+	"os"
+)
+
+var (
+	// ErrorNoRows should be returned whenever no rows were found for the given query
+	ErrorNoRows = pg.ErrNoRows
+)
+
+// DBConfig wraps config for connecting to a database.
+type DBConfig struct {
+	Address  string
+	User     string
+	Database string
+	Password string
+}
+
+// DB wraps a client for a database.
+type DB struct {
+	Client *pg.DB
+}
+
+// Close closes a connection to a database. Once close has been called calling other methods on db will error.
+func (db *DB) Close() {
+	db.Client.Close()
+}
+
+// Migrate runs all migrations found in migrationDir against db.
```

Comment:
Can you document the additional requirements needed to make this work? Thinking specifically in regards to: https://github.com/tozny/client-service/pull/1/files#diff-65d1a23063053a9c8ba9175c4e25dcda